src/Myra/Graphics2D/UI/Desktop.cs

```diff
@@ -19,419 +19,419 @@
 
 namespace Myra.Graphics2D.UI
 {
-	public struct MouseInfo
-	{
-		public Point Position;
-		public bool IsLeftButtonDown;
-		public bool IsMiddleButtonDown;
-		public bool IsRightButtonDown;
-		public float Wheel;
-	}
-
-	public static class Desktop
-	{
-		public const int DoubleClickIntervalInMs = 500;
-
-		public static Func<Rectangle> DefaultBoundsFetcher = () =>
-		{
-			var device = MyraEnvironment.GraphicsDevice;
+    public struct MouseInfo
```

Comment:
Seems like you changed indentation from tabs to spaces. Please, change it back to tabs.